src/test/java/com/jpop/productservice/integration/ProductServiceApplicationIT.java

```diff
@@ -23,18 +32,54 @@
     @Autowired
     TestRestTemplate testRestTemplate;
 
+    @Autowired
+    ProductDetailService productDetailService;
+
+    @TestConfiguration
+    static class ProductServiceApplicationITConfig {
+
+        @Bean
+        public ProductDetailService productDetailService() {
+            return new ProductDetailServiceImpl();
+        }
+    }
+
     @Test
     public void testGetProducts() {
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
         headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
         HttpEntity httpEntity = new HttpEntity(null, headers);
 
-        ResponseEntity<String> responseEntity = testRestTemplate.exchange(createUrl(), HttpMethod.GET, httpEntity, String.class);
+        ResponseEntity<String> responseEntity = testRestTemplate.exchange(createUrl("/api/v1/products/"), HttpMethod.GET, httpEntity, String.class);
         assertEquals("Status code is invalid", 200, responseEntity.getStatusCode().value());
```

Comment:
`status code is invalid`?

Reply:
This message will be shown when there is an assertion failure